src/utils/mapKeys.js

```diff
@@ -0,0 +1,13 @@
+const mapObj = require('map-obj')
```

Comment:
Can't we use `reduce` instead? I find it a bit too much to add a dependency, just for this case.

Reply:
we can (I actually wrote it with reduce first), but it was kind of a long unclear string: `Object.values...reduce...Object.assign`

I thought this was more readable- plus it's very small and has no dependencies itself: https://www.npmjs.com/package/map-obj

Reply:
Wouldn't something like that work?

https://beta.observablehq.com/@nuc/simple-map-obj

I think we should be frugal when adding new dependencies.